media/walkthroughs/discoverAdt/adt-components.md

```diff
@@ -0,0 +1,6 @@
+<!-- markdownlint-disable MD033 MD041 MD013-->
+
+<figure align="center">
+  <img src="all-adt-components.png" alt="image missing" />
+  <figcaption>All the tools inside the Ansible environment package</figcaption>
```

Comment:
Ansible development package?

Reply:
Fixed! The original wireframe used this terminology so it's in both this one and Create an Ansible Env. I'll create a PR to update that one after this.

Reply:
Went ahead and updated that walkthrough in this PR as well, since it's just phrasing and these are closely connected.